speechbrain/pretrained/interfaces.py

```diff
@@ -374,14 +374,14 @@ class EncoderDecoderASR(Pretrained):
 
     Example
     -------
-    >>> from speechbrain.pretrained import EncoderDecoderASR
-    >>> tmpdir = getfixture("tmpdir")
-    >>> asr_model = EncoderDecoderASR.from_hparams(
-    ...     source="speechbrain/asr-crdnn-rnnlm-librispeech",
-    ...     savedir=tmpdir,
-    ... )
-    >>> asr_model.transcribe_file("samples/audio_samples/example2.flac")
-    "MY FATHER HAS REVEALED THE CULPRIT'S NAME"
+    #>>> from speechbrain.pretrained import EncoderDecoderASR
+    #>>> tmpdir = getfixture("tmpdir")
+    #>>> asr_model = EncoderDecoderASR.from_hparams(
```

Comment:
Throws `ImportError: There is no such class as speechbrain.decoders.S2SRNNBeamSearchLM`

Reply:
edits to the pretained interface & the yaml file get it back on track (other edits help to figure out what needs to be done). A revision of the pretrained interface is necessary as well as pretrained yaml files.